src/main/java/net/coderbot/iris/gui/element/ShaderPackListWidget.java

```diff
@@ -86,6 +89,7 @@ public void addEntry(int index, String name) {
 		if (Iris.getIrisConfig().getShaderPackName().equals(name)) {
 			this.setSelected(entry);
 		}
+		packCount++;
```

Comment:
packCount is never reset back to zero when the underlying list is cleared. If it's also possible for the underlying list to be modified in other ways, there are other ways for it to fall out of sync.